### PR TITLE
boot-qemu.py: Handle dtb location shuffle in linux-next

### PR DESCRIPTION
**Pull request by nathanchance**
A refactoring commit adjusted the structure of the dts folder in
`arch/arm` to match `arch/arm64` (vendor sub-directories).

Adjust the `_dtb` member of `QEMURunner` to be a list of potential locations
for the dtbs, so that they can be found before and after this move,
which works for all kernel versions that this script supports.

Link: https://git.kernel.org/next/linux-next/c/724ba6751532055db75992fc6ae21c3e322e94a7

### CODE REVIEW

File: boot-qemu.py [hunk 1]
```diff
@@ -331,7 +332,9 @@ def __init__(self):
 
         self.cmdline.append('earlycon')
 
-        self._dtb = 'aspeed-bmc-opp-palmetto.dtb'
+        self._dtbs = [
+            'aspeed/aspeed-bmc-opp-romulus.dtb', 'aspeed-bmc-opp-romulus.dtb'
+        ]
```
Comment (msfjarvis) [inline boot-qemu.py:337]: Copy-paste fail?

File: boot-qemu.py [hunk 2]
```diff
@@ -72,21 +72,22 @@ def __init__(self):
         self._ram = '512m'
 
     def _find_dtb(self):
-        if not self._dtb:
-            raise RuntimeError('No dtb set?')
+        if not self._dtbs:
```
Comment (msfjarvis) [inline boot-qemu.py:75]: Is an empty list a falsey value?

**Review by msfjarvis — APPROVED** [review]
LGTM, good to go once the dtb name is fixed

File: boot-qemu.py (hunk 1 above)
Comment (nathanchance) [reply boot-qemu.py:337]: Yup, good catch haha!

File: boot-qemu.py (hunk 2 above)
Comment (nathanchance) [reply boot-qemu.py:75]: Yes:

```
>>> not []
True
```

https://docs.python.org/3/library/stdtypes.html#truth-value-testing

**Pull request merged**